dist/wizard-stepper/ds4/wizard-stepper.css

```diff
@@ -0,0 +1,206 @@
+.wizard-stepper__items {
+  -webkit-box-align: center;
+          align-items: center;
+  display: -webkit-box;
+  display: flex;
+  flex-wrap: wrap;
+  -webkit-box-pack: center;
+          justify-content: center;
```

Comment:
Been thinking about this center alignment, and after looking through the mocks, I think we should have left aligned by default, and center aligned as an opt-in modifier, e.g. `wizard-stepper__items--center`.